README.md

```diff
@@ -6,72 +6,148 @@ For guidance on setting up and submitting this assignment, refer to the Marcy la
+friendsManager.addFriend("daniel");
 friendsManager.addFriend(true);
-friendsManager.friends.push('emmaneul');
+friendsManager.friends.push("emmaneul");
 friendsManager.friends.push(42);
 ```
 
 Explain how the code is not consistent or predictable, then provide an example in code that uses closure to make it more consistent and predictable.
 
 ### Response 2
 
+- The code is not consistent or predictable because we see that we are able to add friends through friendsManager using addFriend just like we want and true won’t be passed because we clarified we only want strings. Yet, when we get friends.push we are STILL able to pass friends through! We completely override the fact that we !!!ONLY!!! wanted to add it using addFriend.
+
+```js
+//here we are creating a friendsManager class!
+class friendsManager {
+  //we are starting with *some* friends
+  friends = [“taylor”, “ben”];
+  //declaring our name ^^
+  constructor(name) {
+    //name
+    this.name = name;
+  }
+  //our function that will add a new friends name
+  addFriend(friend) {
+    //we will be pushing our new friend into THIS friends array
+    this.friends.push(friend);
+    //if it works we will console log that it worked
+    console.log(`${friend} has been pushed!`);
+  }
+  //so we can look at our friend array
+  returnFriends() {
+    return [...this.friends];
+  }
+}
+//declaring myself
+const ells = new friendsManager(“ells”);
+//adding gonzalo as my friend
+ells.addFriend(“gonzalo”);
+//returning our friend array back with. . .Gonzalo in it!
+console.log(ells.returnFriends());
+//why cant ben add a friend?
+//ben doesnt have access to our functions and data inside! sorry ben :(
+ben.addFriend(“gonzalo”);
+//even though he’s in the array? yes! because he doesn’t have a instance (‘this’) made for him
+ben.returnFriends();
+```
+
 ## Prompt 3
 
-With OOP in JavaScript, it's possible to use factory functions to achieve encapsulation and re-use them to make objects that look alike. However, factory functions have drawbacks and we often use classes instead. 
+With OOP in JavaScript, it's possible to use factory functions to achieve encapsulation and re-use them to make objects that look alike. However, factory functions have drawbacks and we often use classes instead.
 
 How would you explain to a budding developer what the drawbacks of using factory functions are and why it is better to use classes instead?
 
 ### Response 3
 
+- While with both factory functions and classes allows for users to create objects that hold information that can be repeated and reused.
+  When using a factory function it works as a function that will create not only a new object but recreates the same methods as well. This only leads to an increased and unnecessary use of storage.
+  It would be better for a user to use a class not only for the amount of storage saved but when creating a new instance, the methods will be inherited allowing subclasses to have access and draw from the main class.
```

Comment:
Nice! just use the term "superclass" instead of "main class"

```suggestion
  When using a factory function it works as a function that will create not only a new object but recreates the same methods as well. This only leads to an increased and unnecessary use of storage.
  It would be better for a user to use a class not only for the amount of storage saved but when creating a new instance, the methods will be inherited allowing subclasses to have access and draw from the superclass.
```